agent/proto/agentpb/agent.proto

```diff
@@ -92,12 +92,24 @@ message Probe {
     Failed = 2;
     Terminated = 3;
   }
+  // Severity defines the severity of the probe.
+  enum Severity {
+    // Cleared severity denotes the severity of a running probe
+    Cleared = 0;
```

Comment:
It's not very clear what "cleared" means. No pun intended :) Does it effectively mean "success"?